aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/transfer/ObjectTaggingProvider.java

```diff
@@ -0,0 +1,25 @@
+package com.amazonaws.services.s3.transfer;
+
+import com.amazonaws.services.s3.model.ObjectTagging;
+
+import java.io.File;
+
+/**
+ * This is the callback interface which is used by TransferManager.uploadDirectory and
```

Comment:
minor: Let's change to `{@link TransferManager#uploadDirectory}` so a link is created in the javadocs